crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -255,6 +437,63 @@ def proposals
   api :GET, "/crowbar/:barclamp/1.0/proposals/template",
     "Returns the content of a proposal template"
   param :barclamp, String, desc: "Name of the barclamp", required: true
+  example '
+  {
+    "id": "template-dns",
```

Comment:
as I said there will be follow ups, I really want to avoid those huge PR's. but as this is a significant amount of work which is not necessary for the upgrade process, we have to postpone it.

the card to track this is https://trello.com/c/gs8pbS7M/1170-enhance-api-documentation